schematic/store/synapse.py

```diff
@@ -1584,17 +1794,17 @@ async def format_row_annotations(
             # Otherwise save annotation as approrpriate
             else:
                 if isinstance(anno_v, float) and np.isnan(anno_v):
-                    annos[anno_k] = ""
+                    annos["annotations"]["annotations"][anno_k] = ""
                 elif (
                     isinstance(anno_v, str)
                     and re.fullmatch(csv_list_regex, anno_v)
                     and rule_in_rule_list(
                         "list", dmge.get_node_validation_rules(anno_k)
                     )
                 ):
-                    annos[anno_k] = anno_v.split(",")
+                    annos["annotations"]["annotations"][anno_k] = anno_v.split(",")
                 else:
-                    annos[anno_k] = anno_v
+                    annos["annotations"]["annotations"][anno_k] = anno_v
```

Comment:
I pulled in some changes from: https://github.com/Sage-Bionetworks/schematic/pull/1485 so I could make sure everything was working. I will handle any merge conflicts if that code goes first.